sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -1859,6 +1864,60 @@ public void mergePerson(String leadPersonUuid, String otherPersonUuid, boolean m
 		service.ensurePersisted(leadPerson);
 	}
 
+	private void processPersonAddressMerge(PersonDto leadPersonDto, PersonDto otherPersonDto) {
+		if (differentLocation(leadPersonDto.getAddress(), otherPersonDto.getAddress())) {
+			LocationDto otherAddress = otherPersonDto.getAddress();
+			otherAddress.setAddressType(PersonAddressType.OTHER_ADDRESS);
+			otherAddress.setAddressTypeDetails(I18nProperties.getString(Strings.messagePersonMergedAddressDescription));
+			leadPersonDto.addAddress(otherPersonDto.getAddress());
```

Comment:
I know that this works because of call-by-reference, but it still looks odd to not use `otherAddress` directly here, but access it through otherPersonDto instead

Reply:
changed